### ISSUE TITLE
单卡微调视觉编码器，期待您的快速回复

### ISSUE THREAD

**Issue by CoderXuans** [opendilab/LMDrive#114]
@deepcs233 
你好，我只有一张显卡，我想运行cd vision_encoder bash scripts/train.sh这个命令，我已经修改为
GPU_NUM=1
DATASET_ROOT='/home/syl/LMDrive/autodl-tmp/DATASET_ROOT'

python train_pretrain.py $GPU_NUM $DATASET_ROOT  --dataset carla --train-towns 2  --val-towns 5\
    --train-weathers 0 10 11 12 13 15 16 17 --val-weathers  0 1 2 3\
    --model memfuser_baseline_e1d3_return_feature --sched cosine --epochs 25 --warmup-epochs 5 --lr 0.00075 --batch-size 24  -j 16 --no-prefetcher --eval-metric l1_error \
    --opt adamw --opt-eps 1e-8 --weight-decay 0.05 \
    --scale 0.9 1.1 --saver-decreasing --clip-grad 5 --freeze-num -1 \
    --with-backbone-lr --backbone-lr 0.0003 \
    --multi-view --with-lidar --multi-view-input-size 3 128 128 \
    --smoothed_l1 \
    --experiment memfuser_e1d3 \
    --pretrained \
    但是报错如下：
    usage: train_pretrain.py [-h] [--train-towns TRAIN_TOWNS [TRAIN_TOWNS ...]] [--val-towns VAL_TOWNS [VAL_TOWNS ...]]
                         [--train-weathers TRAIN_WEATHERS [TRAIN_WEATHERS ...]] [--val-weathers VAL_WEATHERS [VAL_WEATHERS ...]] [--saver-decreasing]
                         [--with-lidar] [--with-seg] [--with-depth] [--multi-view] [--multi-view-input-size N N N N N N N N N]
                         [--temporal-frames TEMPORAL_FRAMES] [--freeze-num FREEZE_NUM] [--backbone-lr BACKBONE_LR] [--with-backbone-lr] [--dataset NAME]
                         [--train-split NAME] [--val-split NAME] [--model MODEL] [--pretrained] [--initial-checkpoint PATH] [--resume PATH] [--no-resume-opt]
                         [--num-classes N] [--gp POOL] [--img-size N] [--input-size N N N N N N N N N] [--crop-pct N] [--mean MEAN [MEAN ...]]
                         [--std STD [STD ...]] [--interpolation NAME] [-b N] [-vb N] [--augment-prob AUGMENT_PROB] [--opt OPTIMIZER] [--opt-eps EPSILON]
                         [--opt-betas BETA [BETA ...]] [--momentum M] [--weight-decay WEIGHT_DECAY] [--clip-grad NORM] [--clip-mode CLIP_MODE]
                         [--sched SCHEDULER] [--lr LR] [--lr-noise pct, pct [pct, pct ...]] [--lr-noise-pct PERCENT] [--lr-noise-std STDDEV]
                         [--lr-cycle-mul MULT] [--lr-cycle-limit N] [--warmup-lr LR] [--min-lr LR] [--epochs N] [--epoch-repeats N] [--start-epoch N]
                         [--decay-epochs N] [--warmup-epochs N] [--cooldown-epochs N] [--patience-epochs N] [--decay-rate RATE] [--no-aug]
                         [--scale PCT [PCT ...]] [--ratio RATIO [RATIO ...]] [--hflip HFLIP] [--vflip VFLIP] [--color-jitter PCT] [--aa NAME]
                         [--aug-splits AUG_SPLITS] [--jsd] [--reprob PCT] [--remode REMODE] [--recount RECOUNT] [--resplit] [--smoothing SMOOTHING]
                         [--smoothed_l1] [--train-interpolation TRAIN_INTERPOLATION] [--drop PCT] [--drop-connect PCT] [--drop-path PCT] [--drop-block PCT]
                         [--bn-tf] [--bn-momentum BN_MOMENTUM] [--bn-eps BN_EPS] [--sync-bn] [--dist-bn DIST_BN] [--split-bn] [--model-ema]
                         [--model-ema-force-cpu] [--model-ema-decay MODEL_EMA_DECAY] [--seed S] [--log-interval N] [--recovery-interval N]
                         [--checkpoint-hist N] [-j N] [--save-images] [--amp] [--apex-amp] [--native-amp] [--channels-last] [--pin-mem] [--no-prefetcher]
                         [--output PATH] [--experiment NAME] [--eval-metric EVAL_METRIC] [--tta N] [--local-rank LOCAL_RANK] [--use-multi-epochs-loader]
                         [--torchscript] [--log-wandb]
                         DIR
train_pretrain.py: error: the following arguments are required: DIR
请问你知道如何用单卡训练吗？期待您的快速回复
